nuxeo-retention-package/src/main/resources/install/templates/gcpretention/nuxeo.defaults

```diff
@@ -0,0 +1,14 @@
+## DO NOT EDIT THIS FILE, USE nuxeo.conf  ##
+nuxeo.template.includes=gcpbinaries
+gcpretention.target=.
+
+# mandatory
+nuxeo.gcp.retention.storage.bucket=
+nuxeo.retention.core.binarymanager=org.nuxeo.ecm.core.storage.gcp.GoogleStorageBlobProvider
```

Comment:
Would not it be set by `gcpbinaries`?

Reply:
yes, but I put it explicitly here for the sake of consistency with https://github.com/nuxeo/nuxeo-retention/blob/lts-2025/nuxeo-retention-package/src/main/resources/install/templates/s3retention/nuxeo.defaults#L7